Modules/Libs/QuestieHash.lua

```diff
@@ -83,7 +91,7 @@ function QuestieHash:CompareHashesOfQuestIdList(questIdList)
         else
             local oldHash = questLogHashes[questId]
             if oldHash then
-                local newHash = QuestieHash:GetQuestHash(questId)
+                local newHash = QuestieHash:GetQuestHash(questId) or oldHash
```

Comment:
This leads to wrong return values. Return may be wrong to both ways: Telling there is change when not, and no changes when there was a change.